app/navigation/GuarantorNavigator.scala

```diff
@@ -53,15 +51,25 @@ class GuarantorNavigator @Inject() extends BaseNavigator {
     case GuarantorVatPage => (userAnswers: UserAnswers) =>
       controllers.sections.guarantor.routes.GuarantorAddressController.onPageLoad(userAnswers.ern, userAnswers.lrn, NormalMode)
 
+    case GuarantorAddressPage => (userAnswers: UserAnswers) =>
+      controllers.sections.guarantor.routes.GuarantorCheckAnswersController.onPageLoad(userAnswers.ern, userAnswers.lrn)
+
     case _ =>
```

Comment:
Do we need to add the Check Answers page route here and a test. with a comment saying when tasklist built then redirect to it

Reply:
Updated on https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/94/commits/ddddff079bb90164fbde029f579759419dbd3723